File: PHP/car.php
<?php

class Car{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver  = $driver;
    }

    public function printDataCar(){
        echo "
        license: $this->license, 
        conductor: {$this->driver->name}, 
        Document: {$this->driver->document},
        Numero de Pasajeros: $this->passenger \n <br>";
    }

    public function getPassengers(){
        return $this->passenger;
    }

    public function setPassengers($passenger){
        if ($passenger == 4) {
            $this->passenger = $passenger;
        }else {
            echo "Necesitas asignar 4 pasajeros";
        }
    }
}

File: java/Car.java
public class Car {
    protected String license;
    protected Account driver;
    protected Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDatacar() {
        if(passenger == 4){
            System.out.println(
                "license " + license +
                ", driver " + driver.name +
                "Passengers:" + passenger);
        }
    }

    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger( Integer passenger){
        if(passenger == 4){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    
}
